Baikal/PostEffects/ML/inference_impl.h

```diff
@@ -0,0 +1,28 @@
+#include "Baikal/PostEffects/ML/inference.h"
+
+
+namespace Baikal
+{
+    namespace PostEffects
+    {
+        class InferenceImpl : public Inference
+        {
+        public:
+            InferenceImpl(const std::string& model_path,
+                          size_t width,
+                          size_t height);
```

Comment:
DenoiserInputs also ?

Reply:
At this step we do not need to know that is the contents of the tensor.